Resources/Prototypes/Loadouts/Jobs/Security/detective.yml

```diff
@@ -16,16 +16,6 @@
   id: DetectiveFedoraGrey
   equipment:
     head: ClothingHeadHatFedoraGrey
-  
-# Neck
-- type: loadout
-  id: DetectiveTie
-  equipment: DetectiveTie
-
-- type: startingGear
-  id: DetectiveTie
-  equipment:
-    neck: ClothingNeckTieDet
```

Comment:
why does this just remove the detective tie for no reason

Reply:
that is def not intended oof, will fix